src/interface/src/app/navigation/navigation.component.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { of, BehaviorSubject, Subject } from 'rxjs';
```

Comment:
Do we want to sort imports like this? I've been trying to group the angular imports together and then local imports:

import { ComponentFixture, TestBed } from '@angular/core/testing';
import { NO_ERRORS_SCHEMA } from '@angular/core';
import { RouterTestingModule } from '@angular/router/testing';
import { of, BehaviorSubject, Subject } from 'rxjs';

import { AuthService } from './../auth.service';
import { NavigationComponent } from './navigation.component';

Reply:
I have actually been using the Visual Studio TS Hero plugin to automatically sort imports, but I forgot to run it on this file. Thanks for catching! The plugin matches your preferred grouping, which is nice.